package.json

```diff
@@ -33,21 +33,26 @@
     "android:bundle:staging": "cd android && ./gradlew bundleStagingRelease",
     "android:bundle:prod": "cd android && ./gradlew bundleProdRelease",
     "lint:fix": "eslint --fix src tests",
-    "rename": "./scripts/rename.sh"
+    "rename": "./scripts/rename.sh",
+    "bump": "./rnbv.js"
   },
   "dependencies": {
     "@react-navigation/native": "^6.0.8",
     "@react-navigation/native-stack": "^6.5.2",
-    "react-native-config": "^1.4.6",
     "@tanstack/react-query": "^4.2.1",
     "axios": "^0.27.2",
+    "chalk": "4.1.2",
     "humps": "^2.0.1",
+    "husky": "^8.0.1",
```

Comment:
Why did you add husky as a dependency? We agreed with the team to use Left hook instead. [Here](https://www.notion.so/rootstrap/Pre-Commit-179e78ff2a7f4c568489bc683493b155) you have the explanation